diff --git a/src/app/lib/streamer.js b/src/app/lib/streamer.js
index 32e37832..a5b2cead 100644
--- a/src/app/lib/streamer.js
+++ b/src/app/lib/streamer.js
@@ -404,7 +404,7 @@
                             }, function(err, res) {
                                 if (err) {
                                     win.error('error converting subtitles', err);
-                                    this.streamInfo.set('subFile', null);
+                                    //this.streamInfo.set('subFile', null);
                                     App.vent.trigger('notification:show', new App.Model.Notification({
                                         title: i18n.__('Error converting subtitle'),
                                         body: i18n.__('Try another subtitle or drop one in the player'),
@@ -439,7 +439,7 @@
             }, function(err, res) {
                 if (err) {
                     win.error('error converting subtitles', err);
-                    this.streamInfo.set('subFile', null);
+                    //this.streamInfo.set('subFile', null);
                     App.vent.trigger('notification:show', new App.Model.Notification({
                         title: i18n.__('Error converting subtitle'),
                         body: i18n.__('Try another subtitle or drop one in the player'),
